assets/js/translator.js

```diff
@@ -558,6 +564,25 @@ function populateTranslationList() {
     }
 }
 
+function getSrcNumCols(numSrcCols) {
```

Comment:
> Are we currently forcing them to open up in the directions we want them to? If not, we definitely should. I think there's a Bootstrap data-* setting for it?

Um, for calculations of available space, I am not considering the directions implicitly. Um, I was referring a few links ([this](https://stackoverflow.com/questions/16167178/changing-the-direction-of-a-bootstrap-dropdown) which states that we could use `pull-left` or `pull-right` to achieve the desired directions. Is this something that you are telling about?